packages/protocol/test/bridge/libs/LibBridgeProcess.test.ts

```diff
@@ -0,0 +1,320 @@
+import { expect } from "chai"
+import hre, { ethers } from "hardhat"
+import { Message } from "../../utils/message"
+import { AddressManager, Bridge } from "../../../typechain"
+import { getSlot } from "../../../tasks/utils"
+import * as fs from "fs"
+import * as path from "path"
+const helpers = require("@nomicfoundation/hardhat-network-helpers")
+
+describe("LibBridgeProcess", function () {
+    function getStateSlot() {
+        const buildInfoDir = path.join(
+            __dirname,
+            "../../../artifacts/build-info"
+        )
+        const contractPath =
+            "contracts/test/bridge/libs/TestLibBridgeProcess.sol"
+        const contractName = "TestLibBridgeProcess"
+
+        for (const buildInfoJson of fs.readdirSync(buildInfoDir)) {
+            const { output } = require(path.join(buildInfoDir, buildInfoJson))
+
+            if (!output.contracts[contractPath]) continue
+
+            const slotInfo = output.contracts[contractPath][
+                contractName
+            ].storageLayout.storage.find(({ label }: any) => label === "state")
+
+            if (slotInfo) return Number(slotInfo.slot)
+        }
+
+        throw new Error("TestLibBridgeProcess.state slot number not found")
+    }
+    async function deployLibBridgeProcessFixture() {
+        const [owner, nonOwner, etherVaultOwner] = await ethers.getSigners()
+
+        // slot number of IBridge.State for TestLibBridgeProcess.
+        // mapping destChains is at position 0 (201)
```

Comment:
```suggestion
        // mapping destChains is at position 0
```
Since we fetching the slot number dynamically, there is not need to show the hard-coded slot number in comment here anymore?